.github/actions/deploy-lifecycle-manager/action.yaml

```diff
@@ -0,0 +1,31 @@
+name: Deploy lifecycle-manager
+description: Deploys lifecycle-manager.
+inputs:
+  klm_version_tag:
+    description: The version tag for the KLM image. For example, PR-123.
+    required: true
+  klm_image_repo:
+    description: The repository for the KLM image. For example, dev.
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Deploy LM local testing kustomize
+      working-directory: lifecycle-manager
+      shell: bash
+      run: |
+        maxRetry=5
+        for retry in $(seq 1 $maxRetry)
+        do
+          if make local-deploy-with-watcher IMG=europe-docker.pkg.dev/kyma-project/${{ inputs.klm_image_repo }}/lifecycle-manager:${{ inputs.klm_version_tag }}; then
+            kubectl wait pods -n kcp-system -l app.kubernetes.io/name=lifecycle-manager --for condition=Ready --timeout=90s
```

Comment:
comment: this wait is not there in watcher. Not sure if really needed or if we could omit it.

Reply:
As discussed in person, this is needed and we should definitely keep it to ensure the pods is running in the first retry to have working webhooks